### PR TITLE
Build process : build demo

### PR DESCRIPTION
**Pull request by synbot**
### Acceptance Criteria
1. `npm run demo` builds demo in `/demo`
1. watch for changes and hot-load

### Tasks
- {{list developer tasks here}}

### Additional Notes
- http://davidtheclark.com/building-react-aria-menubutton/
- https://github.com/davidtheclark/react-aria-menubutton

### CODE REVIEW

**Comment by paulstatezny** [comment]
:snowflake: :arrow_up:

File: webpack.config.demo.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    entry: [
+        './demo/bootstrap.js',
+        'webpack/hot/dev-server',
+        'webpack-dev-server/client?http://localhost:9000'
+    ],
+    module: {
+        loaders: [
+            {
+                test    : /\.(jsx|js)$/,
+                loaders : ['babel?loose=all', 'react-hot'],
```
Comment (paulstatezny) [inline webpack.config.demo.js]: What's the point of using loose mode here?

**Comment by paulstatezny** [comment]
:interrobang:

**Comment by zpchavez** [comment]
Optimization I guess. The project I was using as an example did that.

:star:

**Comment by paulstatezny** [comment]
Hmm.

> WARNING
> Please be aware of all possible caveats when enabling loose mode.

http://babeljs.io/docs/advanced/loose/

> As a result of loose mode code will execute faster and be much more readable and comparable to the original but will deviate from the spec in slight ways.

**Comment by zpchavez** [comment]
:star:

**Comment by paulstatezny** [comment]
:pray:

**Comment by pcavagna** [comment]
:hand:

**Comment by pcavagna** [comment]
:+1:

**Pull request merged**